src/Microsoft.Health.Fhir.Api/Controllers/ControlPlaneController.cs

```diff
@@ -0,0 +1,37 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Health.ControlPlane.Core.Features.Rbac;
+
+namespace Microsoft.Health.Fhir.Api.Controllers
+{
+    // TODO: Remove before PR
```

Comment:
Is the comment stil relevant?

Reply:
Yes, but I guess it should be `remove before PR merge`. The goal is that this API shouldn't be exposed anywhere with this code change. It's mainly here for easier manual verification of the refactor.